strings/brain_damage_lines.json

```diff
@@ -91,7 +89,7 @@
 
     "faggot_traitor": [
```

Comment:
remove 90 to 93, and update 6 accordingly

Reply:
Similarly I'd remove `"@pick(semicolon)the ai and borgs are mettacomming I think",`

Reply:
Done. @Pick Var is now listed as traitor and not f-traitor. 6 is updated due to this. AI is metacomming statement got replaced with a false statement about certain eldritch beings having no muscles.